source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -188,7 +188,7 @@ namespace
                             outputName = output.getName();
                         }
                     }
-                    PvtOutput* output = findOutputOrThrow(outputName != EMPTY_TOKEN ? outputName : PvtAttribute::DEFAULT_OUTPUT_NAME, connectedNode);
```

Comment:
This would always force looking for "out" if nothing was specified which is incorrect.